packages/nouns-contracts/tasks/deploy-and-configure.ts

```diff
@@ -1,27 +1,27 @@
-import { task } from 'hardhat/config';
-import { DeployedContract } from './types';
-
-interface ContractRow {
-  Address: string;
-  'Deployment Hash'?: string;
-}
+import { task, types } from 'hardhat/config';
+import { printContractsTable } from './utils';
 
 task('deploy-and-configure', 'Deploy and configure all contracts')
   .addFlag('startAuction', 'Start the first auction upon deployment completion')
   .addFlag('autoDeploy', 'Deploy all contracts without user interaction')
   .addFlag('updateConfigs', 'Write the deployed addresses to the SDK and subgraph configs')
   .addOptionalParam('weth', 'The WETH contract address')
   .addOptionalParam('noundersdao', 'The nounders DAO contract address')
-  .addOptionalParam('auctionTimeBuffer', 'The auction time buffer (seconds)')
+  .addOptionalParam(
+    'auctionTimeBuffer',
+    'The auction time buffer (seconds)',
+    5 * 60 /* 5 minutes */,
+    types.int,
+  )
   .addOptionalParam('auctionReservePrice', 'The auction reserve price (wei)')
   .addOptionalParam(
     'auctionMinIncrementBidPercentage',
     'The auction min increment bid percentage (out of 100)',
   )
-  .addOptionalParam('auctionDuration', 'The auction duration (seconds)')
+  .addOptionalParam('auctionDuration', 'The auction duration (seconds)', 600, types.int)
   .addOptionalParam('timelockDelay', 'The timelock delay (seconds)')
-  .addOptionalParam('votingPeriod', 'The voting period (blocks)')
-  .addOptionalParam('votingDelay', 'The voting delay (blocks)')
+  .addOptionalParam('votingPeriod', 'The voting period (blocks)', 5760, types.int)
+  .addOptionalParam('votingDelay', 'The voting delay (blocks)', 1, types.int)
```

Comment:
Didn't add defaults here so the default values in the `deploy` task would be used. Is there an issue with those? Vaguely remember some type issue.

Reply:
@eladmallel do you remember why these were added?

Reply:
@solimander removed https://github.com/nounsDAO/nouns-monorepo/pull/499/commits/4fd4937e6109a75bab83e4215d35ce4db8334e2b